### PR TITLE
Add CyclOSM layer

### PR DESCRIPTION
**Pull request by frodrigo**
Following the decision of the OWG to add the CyclOSM layer. I try to implement it.

I can adjust the position of the layer if the second position is no right.

### CODE REVIEW

**Comment by mmd-osm** [comment]
Leaflet.osm.js resides in another repo and needs to be updated there: https://github.com/openstreetmap/leaflet-osm

**Comment by pnorman** [comment]
Thanks for this work, I haven't had time to get to it.

Your commit will need to be in leaflet-osm, and should look very much like https://github.com/openstreetmap/leaflet-osm/commit/47b32bae70d24b761887c153853a73a4705fd238. You'll then need a commit in this repo to [update leaflet-osm](https://github.com/openstreetmap/openstreetmap-website/commit/5dfb6432e009b10d3c91f182fa8df72b8a6975c4) and to [add it to the pages](https://github.com/openstreetmap/openstreetmap-website/commit/fb4aa310485cc287a89c3b9c393fb5afeef75bf0).

File: app/assets/javascripts/leaflet.map.js [hunk 1]
```diff
@@ -31,6 +32,13 @@ L.OSM.Map = L.Map.extend({
       name: I18n.t("javascripts.map.base.standard")
     }));
 
+    this.baseLayers.push(new L.OSM.CyclOSM({
+      attribution: copyright + ". " + cyclosm + ". " + terms,
+      code: "Y",
+      keyid: "opnvkarte",
```
Comment (Phyks) [inline app/assets/javascripts/leaflet.map.js:38]: Typo (cyclosm)

Comment (frodrigo) [reply app/assets/javascripts/leaflet.map.js:38]: @Phyks I do not spot what you want to be changed.

Comment (Phyks) [reply app/assets/javascripts/leaflet.map.js:38]: I guess the keyid value (`openvkarte`) is coming from a bad copy paste and should be updated. Isn't it?

Comment (frodrigo) [reply app/assets/javascripts/leaflet.map.js]: Right, fixed, thank.

**Comment by frodrigo** [comment]
I think it is ready. I update leaflet-osm and test all.

**Comment by frodrigo** [comment]
Any news on this ? need more work ?

**Pull request merged**